src/utils/context.js

```diff
@@ -15,6 +15,10 @@ export const initState = {
   },
   writeDemo: {
     theme: Themes('Aarch64'),
+    showHelp: false,
+    fetching: false,
+    usernamePasswordPairs: [],
+    showConsole: false,
```

Comment:
Yeah I'm not sure I see the benefit of this being on the context. Is the thought around consistency? I'm not a big fan of how we're just using global state everywhere if I'm honest

Reply:
Tried to explain in other comments, main reason - keep component code cleaner, easier to understqand and easier to update. We no longer need to pass down the props to the comp we want, we can just read from ctx.